doc/user.rst

```diff
@@ -118,6 +118,9 @@ Input selection options
            only needed if there were multiple continuum imager configurations
            run on this data set.
 
+   The fluxes must be *apparent* fluxes i.e., modulated by the
+   primary beam. That may change in future versions.
```

Comment:
Indeed. And 'apparent' flux isn't really well-defined without a circular beam, so it may eventually be impossible to support apparent densities. But first there is some architecture to get continuum and spectral imagers sharing a beam model.